styles/styles.css

```diff
@@ -0,0 +1,567 @@
+    font-weight: 500;
+}
+
+footer .inner-container {
+    font-size: 14px;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+h1 {
+    margin: var(--default-margin--);
+}
+
+.motto {
+    margin: var(--default-margin--);
+}
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/*outer-containers center the content on the column axis*/
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    /*shorthand for flex-shrink, flex-grow and flex-basis*/
+    flex: 1 1 auto;
+    justify-content: center;
+    z-index: 0;
+}
+
+
+
+/*inner-containers center the content on the row axis*/
+.inner-container {
+    display: flex;
+    flex: 1 1 8rem;
+    flex-direction: row;
+    justify-content: center;
+    position: relative;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    display: flex;
+    flex-direction: column;
+    background: rgb(72, 69, 228);
+    background: linear-gradient(156deg, rgba(72,69,228,1) 30%, rgba(87,34,205,1) 70%);
+    color: var(--header-text-color--);
+}
+
+nav {
+    display: flex;
+    flex: 1 1 auto;
+    padding: 3rem 3rem;
+    gap: 4rem;
+    justify-content: flex-end;
+    text-decoration: none;
+}
+
+
+.header-content {
+    flex: 0 1 42rem;
+    text-align: center;
+    margin: 2rem 1rem;
+}
+
+.squiggle-container {
+    flex: 0 1 4rem;
+    /*shorthand for background*/
+    background: url(../assets/squiggle_line_stripe.svg) bottom / contain no-repeat;
+}
+
+.header-button {
+    background-color: var(--button--color--);
+    padding: 1.5rem 2.5rem;
+    margin: 0.75rem;
+    border: none 50px;
+    border-radius: 50px;
+    color: white;
+    text-transform: uppercase;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.bulletpoint-container {
+
+    flex-direction: column;
+    flex: 0 1 46rem;
+    justify-content: space-between;
+    text-align: center;
+}
+
+.bulletpoints {
+    color: var(--emphasized-text-color--);
+    margin: 8rem 0 3.5rem 0;
+}
+
+/*styles the individual bulletpoints*/
+.bulletpoints h3 {
+    margin: var(--default-margin--);
+}
+
+.contact-us-link {
+    margin-bottom: 8rem;
+}
+
+/* --------------- 4.3  sections ------------------- */
+
+section {
+    position: relative;
+    background-color: var(--background-uneven-main-sections--);
+}
+
+/*responsible for alternating background colours*/
+section:nth-child(even) {
+    background-color: var(--background-even-main-sections--);
+}
+
+/*creates slanted lines for the sections*/
+section::before {
+    position: absolute;
+    content: "";
+    display: block;
+    clip-path: polygon(0 65%, 0 100%, 100% 100%);
+    width: 100%;
+    height: 10rem;
+    background-color: var(--background-uneven-main-sections--);
+    z-index: 1;
+    top: -9.9rem;
```

Comment:
Controleer altijd of je rem waardes overeenkomen met hele pixels. Dit komt namelijk neer op een kommagetal. Dat is niet lekker voor een browser om te renderen (laat staan, schalen)